src/library/Form/propTypes.js

```diff
@@ -12,6 +12,12 @@ import { component } from '../utils/propTypes';
 import { VARIANT } from './constants';
 
 export const formFieldPropTypes = {
+  /* TargetX Custom Props */
+  marginBottom: string,
```

Comment:
In Minerals usual interface for margin, they also support number. I think we should do that here as well.